File: MK/10월 2주차/예상대진표.java
class Solution {
    public int solution(int n, int a, int b) {
        int res=0;

        while (a!=b) {
            res++;
            a = a%2==0?a/2:(a+1)/2;
            b = b%2==0?b/2:(b+1)/2;
        }

        return res;
    }
}

File: DB/10월 3주차/예상대진표.js
function solution(n, a, b) {
  let round = 0;

  // A와 B가 같아지면 경기 끝
  while (a !== b) {
    a = Math.ceil(a / 2);
    b = Math.ceil(b / 2);
    round += 1;
  }

  return round;
}
